draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4743,27 +4718,16 @@ Content-Range: exampleunit 11.2-14.3/25
    is established.  However, most origin servers do not implement CONNECT.
 </t>
 <t>
-   A client sending a CONNECT request &MUST; send the authority form of
-   request-target (<xref target="request.target"/>); i.e., the request-target consists
-   of only the host name and port number of the tunnel destination, separated
-   by a colon. For example,
-</t>
-<artwork type="message/http; msgtype=&#34;request&#34;" x:indent-with="  ">
-CONNECT server.example.com:80 HTTP/1.1
-Host: server.example.com:80
-
-</artwork>
-<t>
-   The recipient proxy can establish a tunnel either by directly connecting to
-   the request-target or, if configured to use another proxy, by forwarding
-   the CONNECT request to the next inbound proxy.
-   Any <x:ref>2xx (Successful)</x:ref> response indicates
-   that the sender (and all inbound proxies) will switch to tunnel mode
-   immediately after the blank line that concludes the successful response's
-   header section; data received after that blank line is from the server
-   identified by the request-target.
-   Any response other than a successful response indicates that the tunnel
-   has not yet been formed and that the connection remains governed by HTTP.
+   The recipient proxy can establish a tunnel either by directly connecting
```

Comment:
But this has removed the necessary requirement that CONNECT does not send a URI in the request target. I hate that exception, always have, but we are stuck with it.